testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1288,3 +1288,66 @@
   node = get_target(host)
   visit("http://#{node.public_ip}:3000/dashboards")
 end
+
+###################################
+## Password Policy navigation steps
+###################################
+
+And(/^I set the minimum password length to "([^"]*)"$/) do |min_length|
+  fill_in 'minLength', with: min_length
+end
+
+And(/^I set the maximum password length to "([^"]*)"$/) do |max_length|
+  fill_in 'maxLength', with: max_length
+end
+
+And(/^I set the special characters list to "([^"]*)"$/) do |characters_list|
+  fill_in 'specialChars', with: characters_list
+end
+
+And(/^I set the maximum allowed occurrence of any character to "([^"]*)"$/) do |max_occurence|
+  fill_in 'maxCharacterOccurrence', with: max_occurence
+end
+
+And(/^I (enable|disable) the following restrictions:$/) do |action, table|
+  restriction_map = {
+    'Require Digits' => 'digitFlag',
+    'Require Lowercase Characters' => 'lowerCharFlag',
+    'Require Uppercase Characters' => 'upperCharFlag',
+    'Require Special Characters' => 'specialCharFlag',
+    'Restrict Characters Occurrences' => 'restrictedOccurrenceFlag',
+    'Restrict Consecutive Characters' => 'consecutiveCharsFlag'
+  }
```

Comment:
Can we move this map to our Constants.rb ?

Reply:
It will be better in a page object ;-). I prefer it there, it's only used in this step, specific to this page. The constants.rb is already big.